### PR TITLE
Doc: Append more doc for addon

### PR DESCRIPTION
**Pull request by yue9944882**
Signed-off-by: yue9944882 <291271447@qq.com>

/assign @qiujian16 @mikeshng

### CODE REVIEW

**Comment by yue9944882** [comment]
Screen Shot 2021-11-17 at 5 58 18 PM 7

btw i think the font size may look a bit awkward over here.. not sure there's any way we can make it look smaller..

File: content/en/concepts/addon.md [hunk 1]
```diff
@@ -5,17 +5,60 @@ weight: 4
...
+addon should consist of two kinds of components:
 
-1. A tool to collect alert events in the managed cluster, and send to the hub cluster.
-2. A network solution that uses the hub to share the network info and establish connection among managed clusters.
-3. A tool to spread security policies to multiple clusters.
+- __Addon Agent__: A kubernetes controller *in the managed cluster* that manages
+  the managed cluster for the hub admins. A typical addon agent is expected to 
+  be working by subscribing the prescriptions (e.g. in forms of CustomResources) 
+  from the hub cluster and then consistently reconcile the state of the managed
+  cluster like an ordinary kubernetes operator does.
```
Comment (qiujian16) [inline content/en/concepts/addon.md:17]: I think it worth also mentioning the addon agent can also connect to a centralized non-kube-api endpoint with mtls

**Review by qiujian16 — COMMENTED** [review]
/approve

File: content/en/concepts/addon.md (hunk 1 above)
Comment (yue9944882) [reply content/en/concepts/addon.md:17]: added contents about custom signers and secret persisting to the addon-framework section at the bottom

**Review by qiujian16 — COMMENTED** [review]
/lgtm

thanks!

**Pull request merged**